kolibri/core/content/api.py

```diff
@@ -160,6 +160,7 @@ class ContentNodeFilter(IdFilter):
     # Use a CharFilter instead a BooleanFilter to be able to avoid cache using
     # different values
     random = CharFilter(method="filter_random")
+    only_topics = BooleanFilter(method="filter_only_topics")
```

Comment:
I think this filter is not needed, it does the same as the `kind` filter.

Reply:
Yes, this makes me realize that we should do the same for `onlyContent`. In the backend, when `onlyContent` is passed, we just use the `kind_in`. Then there is no need to change the `api.py` in the Python side.